pipelines/edx-sysadmin-publish.yml

```diff
@@ -13,6 +18,12 @@ resources:
       name: edx-sysadmin
       uri: https://github.com/mitodl/edx-sysadmin.git
       branch: release
+  - name: edx-sysadmin-status
+    type: github-status
+    source:
+      repository: mitodl/edx-sysadmin
+      access_token: {{repo_github_token}}
```

Comment:
The var syntax uses `(())` rather than `{{}}`. Also, if you can change the var name to something like `github.access_token` that would be helpful so that we can populate a Vault path with that data for the pipeline to pull it out.